roles/deploy/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Initialize
+  deploy_helper: path="{{ project_root }}" state=present
+
+- name: Clone project files
+  git: repo="{{ project_git_repo }}"
+       dest="{{ project_source_path }}"
+       version="{{ project_version }}"
+       accept_hostkey=yes
+
+- name: write unfinished file
+  file: path="{{ project_source_path }}/{{ deploy_helper.unfinished_filename }}" state=touch
+
+- name: Copy files to new build dir
+  command: cp -pr {{ project_source_path }} {{ deploy_helper.new_release_path }}
+
+- name: Move site into root
+  shell: mv {{ deploy_helper.new_release_path }}/{{ project_subtree }}/* {{ deploy_helper.new_release_path }}
+  when: project_subtree is defined
```

Comment:
Thanks. That makes sense. I hadn't imagined the cloned repo would be anything but bedrock alone (vs. including ansible etc.). Thanks for this additional insight into project setup, and for your comments earlier re: committing theme to bedrock repo. I need to digest what you said, and reread  the various discourse threads on that because I feel drawn to modularity of keeping them separate (even though I'm not reusing across multiple sites/projects).